app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ActiveRecord::Base
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] One more important attribute is missing, kindly add `:confirmable` to the devise for user here to help make email confirmation easier.